### PR TITLE
Add about_az.help.txt into en-US culture folder

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description
image

image

Test in sign pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1210337&view=results

Please feel free to try it: https://azposhpreview.blob.core.windows.net/public/Az.Accounts.2.6.1_about.nupkg

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: build.proj [hunk 1]
```diff
@@ -225,7 +230,7 @@
   </Target>
 
   <!-- Everything except Publish -->
-  <Target Name="Full" DependsOnTargets="Clean;Build;OnPremChecks;GenerateHelp;StaticAnalysis;Test" />
+  <Target Name="Full" DependsOnTargets="Clean;Build;OnPremChecks;CopyAboutTopics;GenerateHelp;StaticAnalysis;Test" />
```
[the remark this answers is not in the document]
Comment (isra-fel) [reply build.proj:233]: Or we should at least add comment at the beginning of this file to descriot this target.

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: tools/Docs/GenerateAboutTopicTxt.ps1 [hunk 2]
```diff
@@ -0,0 +1,12 @@
+$aboutFileInputFullName = "C:\Azure\azure-powershell\src\Accounts\Accounts\help\About\about_az.md"
+$aboutFileOutputFullName = "C:\Azure\azure-powershell\src\Accounts\Accounts\help\About\about_az.help.txt"
```
Comment (isra-fel) [inline tools/Docs/GenerateAboutTopicTxt.ps1:2]: We should add comment to explain when and how to use this script.
Also absolution path -> relative path

File: src/Accounts/Accounts/help/About/about_az.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# About topic for Azure PowerShell - about_az 
+
+## TOPIC
+
+about_Az
```
Comment (isra-fel) [inline src/Accounts/Accounts/help/About/about_az.md:5]: What is this section for?

**Comment by BethanyZhou** [comment]
Retried in https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1212233&view=results, can success in build task. Checked Az.Accounts in artifacts, it works as well.

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**